.golangci.yaml

```diff
@@ -148,7 +148,7 @@ linters-settings:
       shadow:
         # Whether to be strict about shadowing; can be noisy.
         # Default: false
-        strict: true
+        strict: false
```

Comment:
why was this changed ?

Reply:
The linter is being really noisy for a typical go error handling.
I think shadowing errors is fine, we shouldn't come up with err2 and err3 variables.